config/src/test/kotlin/org/springframework/security/config/web/server/ServerHttpSecurityDslTests.kt

```diff
@@ -123,4 +129,74 @@ class ServerHttpSecurityDslTests {
             }
         }
     }
+
+    @Test
+    fun `add filter at is applied`() {
```

Comment:
Please make the test names more descriptive.

For example, this test could be renamed to:

`add filter at applies custom at specified filter position`

Reply:
Done.